cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -965,6 +1063,12 @@ def configure_apiserver():
         'DefaultTolerationSeconds'
     ]
 
+    if hookenv.config('authorization-mode').lower() == 'rbac':
+        admission_control.append('NodeRestriction')
+        api_opts.add('authorization-mode', 'Node,RBAC', strict=True)
+    else:
```

Comment:
seems like this should require an exact match of `none`, otherwise `webhook` would enable `AlwaysAllow`, right?